### PR TITLE
Update beaker tests + travis integration

### PR DESCRIPTION
**Pull request by solarkennedy**
This updates the beaker tests to the point where they work again.

@aj-jester this is ready for review. I mostly wanted this in perpetration for integrating your work with installing zips and downgrades.

If we want to be sure that our code can fetch consul from the internet, unzip, verify, downgrade, etc, then we should describe it in code. With these this infrastructure we should be able to do that reproducibly, and if it ever breaks we will know.

Of course you can run them manually yourself if you have docker installed with `bundle exec rake beaker`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,20 +2,22 @@
...
+script: if [ $BEAKER == true ]; then make beaker; else bundle exec rake test; fi
 env:
-#  - PUPPET_VERSION="~> 2.7.0"
   - PUPPET_VERSION="~> 3.1.0"
   - PUPPET_VERSION="~> 3.3.0"
   - PUPPET_VERSION="~> 3.7.4" FUTURE_PARSER=yes STRICT_VARIABLES=yes
   - PUPPET_VERSION="~> 4.0.0"
   - PUPPET_VERSION="~> 4.1.0"
+  - BEAKER=true
+  global:
```
Comment (aj-jester) [inline .travis.yml]: may be the `PUPPET_VERSION=*` need to move into `global` as well? Currently it seems to conflict between the 2 data types; array and hash.

**Comment by aj-jester** [comment]
@solarkennedy niiiiiice! Let me try this again and I'll let you know in a few.

**Comment by solarkennedy** [comment]
I'm going to go ahead and merge this in, as they do pass and work.
We can get more sophisticated as we go and want to ensure certain behaviors.

**Comment by aj-jester** [comment]
@solarkennedy sorry for not getting back quickly. I had time to play around with it earlier this week and think it looks good. I'll play around with it some more this weekend. Thanks.

**Pull request merged**